charts/v2.0.0-alpha.1/azuredisk-csi-driver/values.yaml

```diff
@@ -36,23 +36,23 @@ image:
 
 serviceAccount:
   create: true # When true, service accounts will be created for you. Set to false if you want to use your own.
-  controller: csi-azuredisk-controller-sa # Name of Service Account to be created or used
-  node: csi-azuredisk-node-sa # Name of Service Account to be created or used
+  controller: csi-azuredisk2-controller-sa # Name of Service Account to be created or used
+  node: csi-azuredisk2-node-sa # Name of Service Account to be created or used
   snapshotController: csi-snapshot-controller-sa # Name of Service Account to be created or used
   schedulerExtender: csi-azuredisk-scheduler-extender-sa # Name of Service Account to be created or used
 
 rbac:
   create: true
-  name: azuredisk
+  name: azuredisk2
```

Comment:
We should rely on `--set` to override this values for a side-by-side install. Eventually, we'll want to be able to do an in-place upgrade of the V1 driver.